docs/libcurl/opts/CURLOPT_MAIL_RCPT_IGNORE_INVALID.3

```diff
@@ -0,0 +1,68 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 1998 - 2020, Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at https://curl.haxx.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" **************************************************************************
+.\"
+.TH CURLOPT_MAIL_RCPT_IGNORE_INVALID 3 "16 Jan 2020" "libcurl 7.70.0" "curl_easy_setopt options"
+.SH NAME
+CURLOPT_MAIL_RCPT_IGNORE_INVALID \- ignore invalid recipient addresses during SMTP conversation
+.SH SYNOPSIS
+.nf
+#include <curl/curl.h>
+
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_MAIL_RCPT_IGNORE_INVALID,
+                          long ignore);
+.SH DESCRIPTION
+If \fIignore\fP is set to 1L, ignore the invalid recipients addresses during
+SMTP conversation.
+
+When sending data to multiple recipients, by default curl will abort and return
+failure if one of the recipients indicate failure (on the "RCPT TO" command).
+
+The default behavior can be changed by setting \fIignore\fP to 1L which will
+make curl ignore invalid recipients and proceed with SMTP session normally.
```

Comment:
I think it should clarify that it still requires at least one `RCPT` command to return success.

Reply:
Clarified